main.py

```diff
@@ -275,26 +131,13 @@ def declare_exchange_with_queue(ch: pika.adapters.blocking_connection.BlockingCh
     ch.queue_bind(name, name)
 
 
-def load_kube_config(kubeconfig: str):
-    homeconfig = Path(Path.home(), ".kube/config").absolute()
-    if kubeconfig is not None:
-        kubernetes.config.load_kube_config(kubeconfig)
-    elif homeconfig.exists():
-        kubernetes.config.load_kube_config(str(homeconfig))
-    else:
-        kubernetes.config.load_incluster_config()
-
-
-def call_every(connection, interval, func):
-    def func_every():
-        func()
-        connection.call_later(interval, func_every)
-    connection.call_later(interval, func_every)
-
-
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--debug", action="store_true", help="enable verbose logging")
+    parser.add_argument(
+        "--debug",
+        action="store_true",
+        help="enable verbose logging",
+    )
     parser.add_argument(
         "--upload-publish-name",
         default=getenv("UPLOAD_PUBLISH_NAME", "upload"),
```

Comment:
argument `kubeconfig` is not ued.  remove?

Reply:
Good catch - I'm no longer using any Kubernetes stuff in this service.